doc/man/repoq.1.rst

```diff
@@ -0,0 +1,182 @@
+.. vim: ft=rst sw=2 sts=2 et
+
+==========
+ **repoq**
+==========
+
+------------------------------------------------
+Output repository information for given products
+------------------------------------------------
+
+:Author: Roman Neuhauser <rneuhauser+repose@sigpipe.cz>
+:Date: Feb 04, 2016
+:Copyright: GPL-2.0
```

Comment:
I'll change this to GPL-3.0. Martin changed already the license of the Manual group to User Commands. I think this field refers to repose itself.